### DISCUSSION TITLE
Can not separate two clients' screen

### DISCUSSION THREAD

**Discussion by Seraphli** [rustdesk/rustdesk#7284] [General]
### Bug Description

If two clients connect to one same computer with two monitors, these clients can only see one monitor. If one switch monitor, the other one will also automatically switch monitor, although they have separate identity and connect to the host from different computer.

### How to Reproduce

Need three computer, A, B, C.
A as host and have two monitor, if B and C connect to A at same time, they can only see the same monitor.

### Expected Behavior

The views of different clients can be separated.

### Operating system(s) on local side and remote side

Mac, Manjaro -> Manjaro

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

No need for screenshots.

**Comment by rustdesk** [comment]
1.2.4 may has your expected behvior, but actually I still perfer 1.2.3's behavior, we will may provide an option to let user choose in the future.

Different scenario has different expection, be considerate.

**Reply by Seraphli** [reply]
Thanks. I will try out 1.2.4 when it is released.
